src/pages/CovidStatDetails.js

```diff
@@ -16,69 +16,69 @@ const CovidStatDetails = () => {
         <div className="text-left">
           <FontAwesomeIcon icon={faArrowAltCircleLeft} size="2x" className="text-primary mb-2" onClick={() => navigate(-1)} style={{ cursor: 'pointer' }} />
         </div>
-        <Card className="text-center bg-light p-3" style={{ width: '40rem' }}>
-          <Card.Img src={detailArr.flag} variant="top" style={{ width: '100%' }} />
+        <Card className="text-center bg-light" style={{ width: '40rem' }}>
+          <Card.Img src={detailArr.flag} className="p-3" variant="top" style={{ width: '100%' }} />
           <Card.Body>
             <Card.Title><h1 className="text-primary">{detailArr.country}</h1></Card.Title>
```

Comment:
## Your Details Page
![image](https://user-images.githubusercontent.com/93039359/227446664-527a167c-3bc4-4349-9111-4c03254059cd.png)

## Design's Details Page
![image](https://user-images.githubusercontent.com/93039359/227446705-91a15ccd-a238-402e-9bb7-1bcf1047a295.png)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be great if you make your image and title(country name) horizontally align to match the design guidelines by which your project will look much more organized and attractive 👍